metaissue.go

```diff
@@ -109,29 +112,38 @@ func (s *IssueService) GetCreateMetaWithOptionsWithContextForJira9(ctx context.C
 
 // GetJiraAPIVersion returns a major number of the Jira server version.
 // todo: we need to log the errors here
-func (s *IssueService) GetJiraAPIVersion(ctx context.Context) int {
+func (s *IssueService) isJiraAPI9(ctx context.Context) bool {
 	// serverInfo API returns general information about the current Jira server.
 	// https://docs.atlassian.com/software/jira/docs/api/REST/9.0.0/#serverInfo
 	apiEndpoint := "rest/api/2/serverInfo"
 	req, err := s.client.NewRequestWithContext(ctx, "GET", apiEndpoint, nil)
 	if err != nil {
-		return 0
+		return defaultJiraAPI9
 	}
 	version := new(JiraServerInfo)
 
 	_, err = s.client.Do(req, version)
 	if err != nil {
-		return 0
+		return defaultJiraAPI9
+	}
+
+	// Jira Cloud API returns a bit strange version number now (1001)
+	// now Jira Cloud API doesn't support 9
+	if version.DeploymentType == "Cloud" {
+		return false
 	}
 
 	// versionNumbers contains an array with 3 numbers: major, minor, patch.
 	// we need only a major number
-	return version.VersionNumbers[0]
+	if version.VersionNumbers[0] == 9 {
```

Comment:
use >= for future jira 10.x versions?

Reply:
yes, It makes sense. updated.
my concern was about changing of Jira API for future, but let's try this way.
thanks